pkg/logs/input/kubernetes/launcher.go

```diff
@@ -61,6 +70,14 @@ func NewLauncher(sources *config.LogSources, services *service.Services) (*Launc
 	return launcher, nil
 }
 
+func isAvailable() bool {
+	if _, err := os.Stat(podsDirectoryPath); os.IsNotExist(err) {
```

Comment:
what if the error is a access denied ?

Reply:
Agreed, we probably don't want to launch on any Stat error